src/content/app/genome-browser/components/browser-sidebar-modal/BrowserSidebarModal.tsx

```diff
@@ -27,10 +27,7 @@ import { closeDrawer } from 'src/content/app/genome-browser/state/drawer/drawerS
 
 import SidebarModal from 'src/shared/components/layout/sidebar-modal/SidebarModal';
 
-const browserSidebarModals: Record<
-  string,
-  LazyExoticComponent<() => JSX.Element | null>
-> = {
```

Comment:
Could you search through our codebase for `LazyExoticComponent<() => JSX.Element | null>` and update the two other places where it is used (sidebar in EntityViewer and on the Species page)?